### PR TITLE
Fix search result overflow

### PR DESCRIPTION
**Pull request by kidonng**
Currently, the search result may overflow on small screens:

image

This pull request limits the width of search results, and set a initial scale to prevent similar overflows in the future:

image

To be honest this does not look the best, but it's better than overflowing.

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     {% endif %}
     {% seo title=false %}
     {% feed_meta %}
-    <meta name="viewport" content="width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (MikeMcQuaid) [inline _layouts/base.html:25]: Why is this needed? Can this be scoped just to the list?

File: assets/css/styles.scss [hunk 2]
```diff
@@ -693,4 +693,8 @@ div .algolia-autocomplete {
   }
 }
 
+#algolia-autocomplete-listbox-0 {
```
Comment (MikeMcQuaid) [inline assets/css/styles.scss:696]: Can we use something related to e.g. `search-bar` instead? This feels like an implementation detail that may change.

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks for PR! Some thoughts.

File: assets/css/styles.scss (hunk 2 above)
Comment (kidonng) [reply assets/css/styles.scss:696]: I don't think Algolia is going to change this anytime soon, but I agree we can choose a better selector.

File: _layouts/base.html (hunk 1 above)
Comment (kidonng) [reply _layouts/base.html:25]: This is not needed to fix the bug itself, but to prevent other potential overflows from causing the page to "scale down", like in the first screenshot. Sadly it cannot be scoped.

That said, any overflow needs to be addressed anyway, so it does not matter a lot 😅

Comment (MikeMcQuaid) [reply _layouts/base.html]: @kidonng If it's not needed to fix this bug: can you remove it? Thanks!

Comment (MikeMcQuaid) [reply _layouts/base.html]: Oh, you did, sorry!

**Review by MikeMcQuaid — APPROVED** [review]
Thanks again @kidonng!
